app.conf

```diff
@@ -0,0 +1,23 @@
+server {
+
+	listen 80 default_server;
+	listen [::]:80 default_server;
+
+	root /var/www/app/wordpress;
+
+	index index.php index.html index.htm;
+
+	location / {
+		try_files $uri $uri/ =404;
+	}
+
+	location ~ \.php$ {
+		#include snippets/fastcgi-php.conf;
```

Comment:
do you need all this commented lines?

Reply:
That block gave me a very hard time and that's why I added those comments as "documentation". Just in case I encounter this in the future.

...I could also remove them of course!